src/index.js

```diff
@@ -16,4 +16,10 @@ async function plugin(server) {
 	});
 }
 
-module.exports = fp(plugin, { fastify: "4.x", name: "fastify-disablecache" });
+module.exports = fp(fastifyDisablecache,{
+		fastify: "4.x",
+		name: "fastify-disablecache"
+	}
+);
+module.exports.default = fastifyDisablecache;
```

Comment:
fastifyPlugin is mutating the function itself. so you can just reassign after called fastifyPlugin once.

I usually tried to keep the order of module.exports, then module.exports.default and last module.exports.fastifyPluginName